.github/workflows/single-runner.yml

```diff
@@ -8,31 +8,39 @@ on:
         required: true
         default: dcp_mappluto
       latest:
+        type: boolean
         description: "Tag this version as latest (optional)"
         required: false
         default: "true"
       version:
         description: "The version of the dataset (i.e. 22v2, 21C) if needed (optional)"
         required: false
 
-
-
 jobs:
-  dataloading:
+  setup:
+    uses: ./.github/workflows/setup.yml
+  archive:
     runs-on: ubuntu-20.04
-    env:
-      AWS_S3_ENDPOINT: ${{ secrets.DO_S3_ENDPOINT }}
-      AWS_ACCESS_KEY_ID: ${{ secrets.DO_ACCESS_KEY_ID }}
-      AWS_SECRET_ACCESS_KEY: ${{ secrets.DO_SECRET_ACCESS_KEY }}
-      AWS_S3_BUCKET: edm-recipes
+    needs: setup
     steps:
-      - uses: actions/checkout@v2.3.4
-      - uses: NYCPlanning/action-library-archive@v1.1
-        name: Archive ${{ github.event.inputs.dataset }}
-        with:
-          name: ${{ github.event.inputs.dataset }}
-          latest: ${{ github.event.inputs.latest }}
-          version: ${{ github.event.inputs.version }}
-          s3: true
-          compress: true
-          output_format: pgdump shapefile csv
\ No newline at end of file
+    - name: Setup Docker Buildx
+      uses: docker/setup-buildx-action@v1
+    - name: Download artifact
+      uses: actions/download-artifact@v2
+      with:
+        name: myimage
+        path: /tmp
+    - name: Load Docker image
+      run: |
+        docker load --input /tmp/myimage.tar
+    - name: Archive ${{ github.event.inputs.dataset }}
+      env:
+        latest: ${{ github.event.inputs.latest == 'true' && '--latest' || ' ' }}
+        version: ${{ github.event.inputs.version || ' ' }}
+      run: |
+        echo $latest
+        echo $version
+        poetry run library archive --name ${{ github.event.inputs.dataset }} -o pgdump --s3 --compress $latest $version
```

Comment:
I don't think we always want just the compressed files, what type of output files are pushed to DO (for a specific dataset) when the `single-runner` workflow is run?

Reply:
ahh answer is in the write up

Reply:
@fvankrieken 

sorry I'm not seeing the answer in the write up. just flagging cuz it looks like this changes the `single-runner` output from the 3 formats `pgdump shapefile csv` to only the compressed `pgdump`